pkgs/tools/misc/xvfb-run/default.nix

```diff
@@ -1,29 +1,38 @@
-{ stdenv, fetchurl, makeWrapper, xorgserver, getopt
-, xauth, utillinux, which, fontsConf, gawk, coreutils }:
-let
-  xvfb_run = fetchurl {
-    name = "xvfb-run";
-    # https://git.archlinux.org/svntogit/packages.git/?h=packages/xorg-server
-    url = https://git.archlinux.org/svntogit/packages.git/plain/trunk/xvfb-run?h=packages/xorg-server&id=9cb733cefa92af3fca608fb051d5251160c9bbff;
-    sha256 = "1307mz4nr8ga3qz73i8hbcdphky75rq8lrvfk2zm4kmv6pkbk611";
+{ stdenv, fetchFromGitLab, makeWrapper, xorgserver, getopt, xauth, utillinux, fontsConf, gawk
+, coreutils }:
+
+stdenv.mkDerivation rec {
+  pname = "xvfb-run";
+  version = "1.20.6-1";
+
+  src = fetchFromGitLab {
+    domain = "salsa.debian.org";
+    group = "xorg-team";
+    owner = "xserver";
+    repo = "xorg-server";
+    rev = "xorg-server-2_${version}";
+    sha256 = "03a2x0fds45j7dqsp31ifsr1bm8zl6yac925n0bqdf4sjiyscq20";
   };
-in
-stdenv.mkDerivation {
-  name = "xvfb-run";
-  buildInputs = [makeWrapper];
-  buildCommand = ''
-    mkdir -p $out/bin
-    cp ${xvfb_run} $out/bin/xvfb-run
 
-    chmod a+x $out/bin/xvfb-run
-    patchShebangs $out/bin/xvfb-run
+  phases = [ "unpackPhase" "installPhase" ];
+
+  buildInputs = [ makeWrapper ];
+  installPhase = ''
+    mkdir -p $out/bin $out/share/man/man1
+    install -m 555 debian/local/xvfb-run $out/bin/xvfb-run
+    install debian/local/xvfb-run.1 $out/share/man/man1/xvfb-run.1
+
+    patchShebangsAuto
     wrapProgram $out/bin/xvfb-run \
       --set FONTCONFIG_FILE "${fontsConf}" \
-      --prefix PATH : ${stdenv.lib.makeBinPath [ getopt xorgserver xauth which utillinux gawk coreutils ]}
+      --prefix PATH : ${stdenv.lib.makeBinPath [ getopt xorgserver xauth utillinux gawk coreutils ]}
   '';
 
   meta = with stdenv.lib; {
+    description = "A wrapper for the Xvfb command which simplifies the task of running commands";
+    homepage = src.meta.homepage +  "debian/local";
+    license = licenses.mit;
```

Comment:
Where did you find this? Are you sure that is not just a license of Xorg? The manpage contains `gpl2Plus`.

Reply:
Since the repository is a fork of the Xorg tree and the script itself has no license notice, it is distributed under the Xorg license, right?

The Xorg license is `a slight variant of the common MIT license form`.

The `xorg.xorgserver` package currently has no license (see https://nixos.org/nixos/packages.html?attr=xorg.xorgserver&channel=nixos-19.09) but the `xwayland` package build from the same source is declared a `licenses.mit` (see https://nixos.org/nixos/packages.html?attr=xwayland&channel=nixos-19.09), so I thought `licenses.mit` would be correct.

I didn't notice the GPL license in the manpage.

Which license is the correct one now?

Reply:
Since the script itself has no license information, the general license for the repo applies.

https://salsa.debian.org/xorg-team/xserver/xorg-server/blob/debian-unstable/COPYING

But the manpage is GPL ;)

Which seems messed up and unintended... maybe open an issue upstream to clearify.